From cf84060f101f0566f5fdc2e85b096d2331034df0 Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Tue, 19 Nov 2024 10:46:11 +0100
Subject: [PATCH 1371/1695] dt-bindings: scmi: rename scmi regulators by
 regulator@0

Use a scmi regulator node name coherent with other scmi resources:
protocol@17{
	regulators {
		regulator@0 {
		};
		[...]
	};
};

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 92d83c904a06375ebc244ef13bcb66b55edf46b7 dt-bindings: scmi: rename scmi regulators by regulator@0]
Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: Iec96c844df855d0e7024258ac81a8526d0c31735
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/419477
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 documentation/devicetree/bindings/scmi/scmi.yaml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/documentation/devicetree/bindings/scmi/scmi.yaml b/documentation/devicetree/bindings/scmi/scmi.yaml
index 7f0d04603..65b90dbba 100644
--- a/documentation/devicetree/bindings/scmi/scmi.yaml
+++ b/documentation/devicetree/bindings/scmi/scmi.yaml
@@ -199,7 +199,7 @@ patternProperties:
               "#size-cells":
                 const: 0
             patternProperties:
-              "^voltd@[0-9a-f]+$":
+              "^regulator@[0-9a-f]+$":
                 type: object
                 properties:
                   reg:
-- 
2.47.2

